src/launcher/StracciatellaLauncher.cc

```diff
@@ -98,75 +99,70 @@ le from the mod with the highest priority is used.");
-          } // Fl_Choice* scalingModeChoice
-          o->end();
-        } // Fl_Group* o
-        { Fl_Group* o = new Fl_Group(10, 170, 500, 45);
-          { resolutionXInput = new Fl_Value_Input(20, 185, 75, 30, "Resolution:");
+        { Fl_Group* o = new Fl_Group(10, 115, 500, 52);
+          o->tooltip("Specify the resolution for tactical screen rendering. Texts and sprites becom\
+e very small at higher internal resolutions. Note that menus and the strategic\
+ map screen are always rendered at 640x480.");
+          { resolutionXInput = new Fl_Value_Input(20, 130, 75, 30, "Internal Resolution:");
             resolutionXInput->minimum(640);
             resolutionXInput->maximum(0);
             resolutionXInput->value(640);
             resolutionXInput->align(Fl_Align(FL_ALIGN_TOP_LEFT));
           } // Fl_Value_Input* resolutionXInput
-          { new Fl_Box(95, 185, 20, 30, "x");
+          { new Fl_Box(95, 130, 20, 30, "x");
           } // Fl_Box* o
-          { resolutionYInput = new Fl_Value_Input(115, 185, 75, 30);
+          { resolutionYInput = new Fl_Value_Input(115, 130, 75, 30);
             resolutionYInput->labeltype(FL_NO_LABEL);
             resolutionYInput->minimum(480);
             resolutionYInput->maximum(0);
             resolutionYInput->value(480);
             resolutionYInput->align(Fl_Align(FL_ALIGN_TOP_LEFT));
           } // Fl_Value_Input* resolutionYInput
-          { predefinedResolutionMenuButton = new Fl_Menu_Button(210, 185, 110, 30, "Predefined");
+          { predefinedResolutionMenuButton = new Fl_Menu_Button(210, 130, 110, 30, "Predefined");
           } // Fl_Menu_Button* predefinedResolutionMenuButton
-          { Fl_Group* o = new Fl_Group(415, 170, 30, 30, "resizable");
+          { Fl_Group* o = new Fl_Group(415, 115, 30, 30, "resizable");
             o->labeltype(FL_NO_LABEL);
             o->align(Fl_Align(FL_ALIGN_TOP|FL_ALIGN_INSIDE));
             o->end();
             Fl_Group::current()->resizable(o);
           } // Fl_Group* o
-          { invalidResolutionLabel = new Fl_Box(340, 185, 160, 30, "invalid!");
+          { invalidResolutionLabel = new Fl_Box(340, 130, 160, 30, "invalid!");
             invalidResolutionLabel->tooltip("the resolution must be at least 640x480, the original game resolution");
             invalidResolutionLabel->labelfont(1);
             invalidResolutionLabel->labelcolor((Fl_Color)1);
             invalidResolutionLabel->align(Fl_Align(FL_ALIGN_LEFT|FL_ALIGN_INSIDE));
           } // Fl_Box* invalidResolutionLabel
           o->end();
         } // Fl_Group* o
+        { Fl_Group* o = new Fl_Group(10, 170, 500, 55);
+          o->tooltip("Choose a scaling mode to stretch the game video to fit the screen. Linear and\
+ Near-Perfect modes stretch the video to match any window sizes, but the image\
+ may look blurred. Pixel-Perfect can only scale up by integer (1x, 2x and so o\
+n) but gives sharp images.");
```

Comment:
Not sure how much space there is in the tooltip, but it might make sense to add that you need to enable fullscreen for pixel-perfect scaling and use the "low-resolution" that will be scaled up.

Reply:
You don't need full screen mode for it. Just resize the window until it's large enough.

We should later make a 2x/3x button in the scaling option, instead of having the code to calculate on window resize. That should be much less confusing